### DISCUSSION TITLE
PoPageDynamicTableComponent - URL inadequada

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1138] [Tire suas dúvidas] [answered]
Olá,

Estou tentando usar o atributo "range" na busca avançada do "Page Dynamic Table", porem creio que a URL está sendo gerada de forma inadequada. Segue abaixo o objeto "fields" e a URL gerada.

```
  readonly fields: Array<PoPageDynamicTableFilters> = [
    { property: 'id', label: 'Id', key: true, visible: false },
    { property: 'date', label: 'Data', type: 'date', filter: true, range: true },
  ];

```

[http://localhost:4200/api/visitors?page=1&pageSize=10&date=%5Bobject%20Object%5D](http://localhost:4200/api/visitors?page=1&pageSize=10&date=%5Bobject%20Object%5D)

**Comment by alinelariguet** [comment, marked as answer]
Olá @wsteixeira , vamos analisar e logo daremos retorno.
